sb600spi.c

```diff
@@ -332,6 +333,17 @@ static int spi100_spi_send_command(const struct flashctx *flash, unsigned int wr
 	}
 	msg_pspew("\n");
 
+	mmio_writeb(mmio_readb(sb600_spibar + 0x0e) | 0x80, sb600_spibar + 0x0e);
```

Comment:
Please add a comment what this does and why. AFAIK this bit is reserved.

Reply:
This is not supposed to be here. Some leftover from my testing. WIll remove.

Reply:
https://github.com/Dasharo/flashrom/pull/14/files#diff-718efe111fd056eb79853f7ea895d085fea98e8af488abfdb5aee6aa48d4e4aeR1467